src/pages/admin/store/pickup/[uuid].tsx

```diff
@@ -0,0 +1,106 @@
+import { PickupOrdersDisplay } from '@/components/admin/store';
+import { PickupEventStatus } from '@/components/admin/store/PickupEventCard';
+import { Typography } from '@/components/common';
+import { EventCard } from '@/components/events';
+import { config } from '@/lib';
+import { StoreAPI } from '@/lib/api';
+import withAccessType from '@/lib/hoc/withAccessType';
+import { CookieService, PermissionService } from '@/lib/services';
+import { PublicOrderPickupEvent } from '@/lib/types/apiResponses';
+import { CookieType } from '@/lib/types/enums';
+import { formatEventDate } from '@/lib/utils';
+import styles from '@/styles/pages/StorePickupEventDetailsPage.module.scss';
+import { GetServerSideProps } from 'next';
+import Link from 'next/link';
+import { useState } from 'react';
+
+interface PickupEventDetailsPageProps {
+  pickupEvent: PublicOrderPickupEvent;
+}
+
+const PickupEventDetailsPage = ({ pickupEvent }: PickupEventDetailsPageProps) => {
+  const { status, title, start, end, orderLimit, description, linkedEvent, orders } = pickupEvent;
+  const [ordersView, setOrdersView] = useState<'fulfill' | 'prepare'>('fulfill');
+
+  return (
+    <div className={styles.page}>
+      <Link href={config.admin.store.pickup} className={styles.back}>
+        {'< Back to dashboard'}
+      </Link>
+      <div className={styles.container}>
+        <div className={styles.pickupDetails}>
+          {linkedEvent ? (
+            <EventCard event={linkedEvent} attended={false} hideInfo />
+          ) : (
+            <Typography variant="h3/regular" className={styles.noEvent}>
+              *Not linked to an event on the membership portal
+            </Typography>
+          )}
+          <div>
+            <PickupEventStatus status={status} variant="h3/bold" />
+            <Typography variant="h1/bold">{title}</Typography>
+            <Typography variant="h4/regular" suppressHydrationWarning>
+              {formatEventDate(start, end, true)}
+            </Typography>
+            <Typography variant="h4/regular">{`Max Orders: ${orderLimit}`}</Typography>
+            <Typography variant="h4/regular">{description}</Typography>
```

Comment:
![image](https://github.com/acmucsd/membership-portal-ui-v2/assets/22133785/447e74c4-8ea6-4cbd-aef6-6e8226d2e992)

you should add `word-break: break-word;` and `white-space: pre-wrap;` here I think

I just set it to a long word followed by two line breaks and "hey"

```js
$0.nodeValue = 'fdgdfgfdfd'.repeat(400) + '\n\nhey'
```